hathor/manager.py

```diff
@@ -215,6 +215,11 @@ def start(self) -> None:
         self.is_started = True
 
         self.log.info('start manager', network=self.network)
+        self.log.info(
+            'Settings configuration',
+            ws_max_subs_addrs_conn=settings.WS_MAX_SUBS_ADDRS_CONN,
+            ws_max_subs_addrs_empty=settings.WS_MAX_SUBS_ADDRS_EMPTY
+        )
```

Comment:
I think this is weird. Can't we make this a debug log? Or what if we log the settings module path instead, that way anyone using custom settings will know if their settings is actually being used.